docs/examples/aws_ebs_volume_idempotence.rb

```diff
@@ -0,0 +1,34 @@
+require 'chef/provisioning/aws_driver'
+
+with_driver 'aws::us-west-2'
+
+key = aws_key_pair 'ref-key-pair-ebs'
+key.run_action(:create)
+
+ref_machine1 = machine 'ref-machine-1'
+ref_machine1.run_action(:converge)
+
+ebs_volume = aws_ebs_volume 'ref-volume-ebs' do
+  action :nothing
+  machine ref_machine1.name
+  device '/dev/xvdf'
+  availability_zone 'us-west-2a'
+  size 1
+end
+
+ebs_volume.run_action(:create)
+ebs_volume.run_action(:create) # up to date
+ebs_volume.run_action(:attach)
+ebs_volume.run_action(:attach) # up to date
+ebs_volume.run_action(:detach)
+ebs_volume.run_action(:detach) # up to date
+ebs_volume.run_action(:delete)
+ebs_volume.run_action(:delete) # up to date
```

Comment:
Although now that I think about it, you should be able to do my previous code block now and have it work without resource cloning.  What was the problem you were seeing when trying to do that?